test/groovy/util/JenkinsBatGitRule.groovy

```diff
@@ -0,0 +1,63 @@
+package util
```

Comment:
This is not yet relevant for the current codebase since it seems to relate to the the bats testing framework. Might get relevant in the future though.

Reply:
I kicked it out. Let's take it on board in case it gets relevant.